pkg/pillar/hardware/smart.go

```diff
@@ -125,7 +130,17 @@ func GetInfoFromSATAdisk(diskName string) (*types.DiskSmartInfo, error) {
 	}
 
 	diskInfo.ModelNumber = devIdentify.ModelNumber()
+	log.Noticef("Model number before massaging %s", diskInfo.ModelNumber)
```

Comment:
Can you check how this %s will print if there are non-printable characters in the 0x00-0x1f range? I wonder whether we would be more information if you instead printed this (and the other three in this file) as a byte slice to make sure we see all of the bytes.

Reply:
Makes sense, will do